lib/pod/try_settings.rb

```diff
@@ -0,0 +1,47 @@
+module Pod
+  class TrySettings
+    attr_accessor :pre_install_commands, :project_path
+
+    # Creates a TrySettings instance based on a folder path
+    #
+    def self.settings_from_folder(path)
+      settings_path = Pathname.new(path) + '.cocoapods.yml'
+      return TrySettings.new unless File.exist? settings_path
+
+      settings = YAML.load(File.read(settings_path))
+      try_settings = TrySettings.new
+      try_settings.pre_install_commands = Array(settings['try']['install']['pre'])
```

Comment:
Wouldn't this crash if we have a settings file and it doesn't have an install pre setting?